Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -121,7 +131,7 @@ export function CreateOptionsV2(props: CreateOptionsProps) {
         dialogType={DialogTypes.Customer}
         onDismiss={onDismiss}
       >
-        <ChoiceGroup required defaultSelectedKey="B" options={options} onChange={handleChange} />
+        <ChoiceGroup required defaultSelectedKey="B" options={getOptions()} onChange={handleChange} />
```

Comment:
what is defaultSelectedKey B?

Reply:
why this change?? has nothing to do with telemetry

Reply:
seems selected key was off regardless of this PR, I have changed it to be a valid key value given the options.

Reply:
Ben I have made this to be not a function